index.coffee

```diff
@@ -63,7 +67,10 @@ initApp = ->
 initPassport = ->
   passport.serializeUser serializeUser
   passport.deserializeUser deserializeUser
-  passport.use new LocalStrategy { usernameField: 'email', passReqToCallback: true }, artsyCallback
+  passport.use new LocalStrategy(
+    { usernameField: 'email', passReqToCallback: true }
+    artsyCallback
```

Comment:
These are all just breaking before 80 characters btw... nothing functionally should have changed.